src/sentry/integrations/jira/utils/create_issue_schema_transformers.py

```diff
@@ -0,0 +1,137 @@
+from collections.abc import Callable, Iterable, Mapping
+from typing import Any, TypeVar
+
+# A list of common builtin custom field types for Jira for easy reference.
+JIRA_CUSTOM_FIELD_TYPES = {
+    "select": "com.atlassian.jira.plugin.system.customfieldtypes:select",
+    "textarea": "com.atlassian.jira.plugin.system.customfieldtypes:textarea",
+    "multiuserpicker": "com.atlassian.jira.plugin.system.customfieldtypes:multiuserpicker",
+    "tempo_account": "com.tempoplugin.tempo-accounts:accounts.customfield",
+    "sprint": "com.pyxis.greenhopper.jira:gh-sprint",
+    "epic": "com.pyxis.greenhopper.jira:gh-epic-link",
+    "team": "com.atlassian.jira.plugin.system.customfieldtypes:atlassian-team",
+    "rank": "com.pyxis.greenhopper.jira:gh-lexo-rank",
+    "development": "com.atlassian.jira.plugins.jira-development-integration-plugin:devsummarycf",
+}
```

Comment:
I've temporarily moved this here to avoid cycles in our imports, but this is probably going into the create_issue_metadata model file.